settings.gradle.kts

```diff
@@ -26,6 +28,7 @@ pluginManagement {
 
     plugins {
         id("com.google.devtools.ksp") version "1.9.0-1.0.11"
+        id("org.eclipse.kuksa.vss-processor-plugin") // Always take the newest version since it's locally included
```

Comment:
As long as we don't define a version this will work. However gradle will complain with a version because they should be defined outside the app/build.gradle.kts FYI. Changed this because I don't think we will ever define a version here.